docs/source/science/NISAR/Simulated_NISAR.ipynb

```diff
@@ -0,0 +1,11846 @@
+{
```

Comment:
<p>Although this works, we shouldn't need to do this. We can ever so slightly simplify things (although this is already pretty simple) by deleting this line and getting an s3fs.S3FileSystem instance directly from earthaccess. See my comment on the next code cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>